GHzDACs/ghz_fpga_server.py

```diff
@@ -607,13 +612,15 @@ def run(self, runners, reps, setupPkts, setupState, sync, getTimingData,
         # setupState is a set
         setupPkts.extend(pkt for pkt, state in boardSetupPkts)
         setupState.update(state for pkt, state in boardSetupPkts)
-        
+
         try:
             yield self.pipeSemaphore.acquire()
+            logging.info('pipe semaphore acquired')
```

Comment:
Should these be debug level instead? Logging can slow things down with e.g. synchronous writes to files or the console, so I'd be hesitant to put too much of it here by default.